### ISSUE TITLE
A lighter distro ?

### ISSUE THREAD

**Issue by abda11ah** [MRColorR/bitping#2]
First of all I'd like to thank you for this work because the official image didn't want to work on my raspberry pi.

It's a good image, works well but seems a bit heavy to me.
Most docker images are only 40MB. And as I have a raspberryPi, resources are quite limited.
I suggest you build it on an alpine-linux or dietpi distro (to stay apt-get compatible).

**Comment by MRColorR** [comment]
Hello,

Thanks for your feedback and for using this Docker image👍. I acknowledge the importance of a lightweight image for low resources devices like Raspberry Pi. I've noted your suggestion about alpine-linux or dietpi and will revisit this idea in future updates.

In the past i tried using a small alpine image but encountered some problems with the spawned processes. I'll try it again in the next few weeks. 

Thank you for your support

**Comment by MRColorR** [comment]
After receiving your message, I took a closer look at the current image setup. I initially based the image on a slimmed-down version of Debian, which contributes approximately 30-40 MB to the image size. However, the necessity to incorporate all the Bitping dependencies does indeed increase the total image size.

In response to your feedback, I delved deeper into potential optimizations and am pleased to report that I've pushed an updated version of the Dockerfile. This latest iteration includes further slimming measures by removing unnecessary files, components and dependencies, which should reduce the image size of about 20% 👍 .
